Server/App/model/discover.py

```diff
@@ -0,0 +1,42 @@
+import json
+
+from model.deck import Deck
+from utils.database import DatabaseConnector
+from utils.serializer import serializer
+
+
+class Discover(object):
+    def __init__(self, user_id, connector=None):
+        self.user_id = user_id
+        self.connector = connector or DatabaseConnector()
+
+    def fetch(self, category, page=1):
+        deck_uuids = self.connector.call_procedure('FETCH_DECK_UUIDS', category, page)
+        results = []
+        for deck_uuid in deck_uuids:
+            results.append(Deck(deck_uuid['uuid'], connector=self.connector).get_metadata(self.user_id))
```

Comment:
Seems like a lot of individual requests for decks in the new scheme, instead of just getting them in mass through joins. Seems like this could get rather inefficient with larger numbers?